test/e2e/tests/notifications/mocks.ts

```diff
@@ -34,6 +34,19 @@ export async function mockNotificationServices(server: Mockttp) {
   mockAPICall(server, await StorageMocks.getMockUserStorageGetResponse());
   mockAPICall(server, await StorageMocks.getMockUserStoragePutResponse());
 
+  // TODO - add better mock responses for other Profile Sync features
+  // (Account Sync, Network Sync, ...)
```

Comment:
Yeah, we are currently mocking these API responses to fail fast. This ensures that the syncing features won't get in the way with existing e2e tests.